src/content/docs/en/guides/cms/datocms.mdx

```diff
@@ -0,0 +1,202 @@
+---
+title: DatoCMS & Astro
+description: Add content to your Astro project using DatoCMS
+type: cms
+stub: false
+service: DatoCMS
+i18nReady: true
+---
+import FileTree from '~/components/FileTree.astro'
+
+[DatoCMS](https://datocms.com/) is the headless CMS for the modern web.
+
+## Integrating with Astro
+
+In this guide, you will learn how to get the data of a single page from DatoCMS using Astro.
+
+## Prerequisites
+
+To get started, you will need to have the following:
+
+- **A DatoCMS Account and project** - if you don't have an account, you can [sign up for a free account](https://dashboard.datocms.com/signup)
+- **The read-only API Key for your DatoCMS Project** - you can find it in the admin dashboard of your Project, under "Settings" > "API Tokens"
+
+## Setting up the credentials
+
+For safety, it is best practice to save your DatoCMS API key in an **environment variable**.
+Let's start by creating a new file named `.env` in the root of your Astro project. In there, you can create a new variable for our DatoCMS API Key as follows: 
+
+```ini title=".env"
+DATOCMS_API_KEY=YOUR_API_KEY
+```
+
+If you're using TypeScript for this project, you can declare the typing of our environment variable in the `env.d.ts` file in the `src` folder. If you didn't opt-in for TypeScript usage when setting up your project, you can create the file yourself.
+
+```ts title="src/env.d.ts"
+interface ImportMetaEnv {
+  readonly DATOCMS_API_KEY: string;
+}
+```
+
+:::tip
+Read more about [using environment variables](/en/guides/environment-variables/) and `.env` files in Astro.
+:::
+
+Your root directory should now include these new files:
+
+<FileTree title="Project Structure">
+- src/
+  - **env.d.ts**
+- **.env**
+- astro.config.mjs
+- package.json
+</FileTree>
+
+## Create a Model in DatoCMS
+
+Before you can continue, you'll need to create a homepage in DatoCMS. Go to the admin dashboard of your Project, and navigate to the "Settings" tab. Click on "Models", and create a new Model called "Home". Make sure to toggle the "Single instance" toggle. In the model, add a new text field for a title.
+
+Navigate to the "Content" tab and click on your newly created Homepage. You can now add a title. Save the page, and continue.
```

Comment:
A "model" is DatoCMS' take on a page basically. It's a template for a type of page that contains a set of blocks defined by the user. I'd personally just stick to "model" in this case because that's how it's named in the DatoCMS Admin Dashboard.